client/signup/utils.js

```diff
@@ -144,11 +144,35 @@ export function getThemeForDesignType( designType ) {
 	}
 }
 
+export function getThemeForSiteType( siteType ) {
+	if ( 'blogger' === siteType ) {
```

Comment:
Since `'pub/independent-publisher-2'` is the default return value, do we need this condition?